src/development/accessibility-and-localization/accessibility.md

```diff
@@ -144,7 +144,34 @@ widgets to offer a dramatically more accessible experience.
 
 <iframe width="560" height="315" src="{{site.youtube-site}}/embed/bWbBgbmAdQs" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 
-## Testing accessibility on web:
+## Testing accessibility on mobile
+
+You can use Flutter's <a href="https://api.flutter.dev/flutter/flutter_test/AccessibilityGuideline-class.html">Accessibility Guideline API</a> to test if your mobile app's UI meets the accessibility recommendations for text contrast, target size, and target labels. 
+
+The example below shows how to use the Guideline API on <a href="https://docs.flutter.dev/get-started/codelab/?tab=talkback">Startup Name Generator</a>, the app built as part of the codelab on creating your first Flutter app. Each list tile on the app's main screen is tappable target with text represented in 18 point. You can add the tests provided by the Guideline API in `test/widget_test.dart` of your app directory.
+
+<?code-excerpt path-base="codelabs/startup_namer/step3_stateful_widget"?>
+<?code-excerpt "test/widget_test.dart" title indent-by="2"?>
```

Comment:
It looks like this code excerpt is out of date. 

When I run [`./tool/refresh-code-excerpts.sh`](examples/codelabs/startup_namer/step3_stateful_widget/test/widget_test.dart), it replaces the entire snippet with the contents from the file at : `examples/codelabs/startup_namer/step3_stateful_widget/test/widget_test.dart`

Mauli, you'll have to integrate that excerpt into that widget_test.dart file in the codelabs repo (cc @domesticmouse), land it, and then pull in the updated codelab submodule into this PR. The refresh code excerpts should be happy when that's completed.

Reply:
The updated codelab that should land in the next update cycle will contain @pandeymauli's updated `widget_test.dart`, so hopefully everything just works?